store/mongo/migration_1_0_0.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	IndexNameSettingsGet = "settings get"
+	IndexNameIntegrationsGet = "integrations get"
```

Comment:
Is it safe to use spaces in the mongodb index name?

Reply:
I believe [it's safe](https://docs.mongodb.com/manual/indexes/#index-names) but do we have any convention for naming indexes? I should probably change the name anyway since it's not very descriptive of the usage.

Reply:
I changed the name to reflect the index keys. I also added keys `provider` and `_id` to provide means of sorting the result.